proc-macros/Cargo.toml

```
@@ -13,4 +13,5 @@ proc-macro = true
 Inflector = "0.11.4"
 proc-macro2 = "1.0"
 quote = "1.0"
-syn = { version = "1.0", features = ["full", "extra-traits", "parsing", "printing", "proc-macro", "visit"] }
\ No newline at end of file
+syn = { version = "1.0", features = ["full", "extra-traits", "parsing", "printing", "proc-macro", "visit"] }
```

```suggestion
syn = { version = "1.0", features = ["full", "extra-traits", "visit"] }
```

"parsing", "printing" and  "proc-macro" are enabled by default I think. Alternatively we can be explicit and do:

`syn = { version = "1.0", features = ["full", "extra-traits", "parsing", "printing", "proc-macro", "visit"], default-features = false }` (but maybe we need `clone-impls` too then?

---

cool, removed the unused ones.